release/internal/hashreleaseserver/config.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hashreleaseserver
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Config holds the configuration for an SSH connection
+type Config struct {
+	// Host is the host for the SSH connection
+	Host string `envconfig:"DOCS_HOST"`
```

Comment:
Save this for another PR, but we're renaming this away from docs so the env vars should be modifies as well.